components/x-gift-article/src/GiftArticle.scss

```diff
@@ -35,6 +36,13 @@ $o-typography-is-silent: true;
 
 	.radio-button-section {
 		margin-bottom: 12px;
+
+
+	}
+
+	.share-option-title {
+		color: blue;
```

Comment:
Is there a specific blue from the `o-colors` palette that can be used?

Reply:
@andygout was using that to debug my css should have removed before commit. Lesson in not using hacky methods to debug things 😬